src/test/java/com/microsoft/sqlserver/jdbc/AlwaysEncrypted/BulkCopySendTemporalDataTypesAsStringAETest.java

```diff
@@ -0,0 +1,172 @@
+/*

```

Comment:
Please format all the files.

Reply:
This file is formatted. Do you see something that should've been formatted? My formatter doesn't change anything in this file.

Reply:
The comments seem to be off.

Reply:
Which comment for example?